src/sentry/deletions/defaults/group.py

```diff
@@ -57,6 +57,7 @@ def chunk(self):
 
         # Remove EventAttachment and UserReport
         event_ids = [event.event_id for event in events]
+
```

Comment:
Someone's padding their line contribution count.

Reply:
Whoops. I didn't mean to change this file.